Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version: 5.8
+
+import PackageDescription
+
+private let packageName = "SimpleNetworkService"
+private let networkService = "NetworkService"
+private let networkServiceTest = networkService + "Tests"
+
+let package = Package(
+    name: packageName,
+    platforms: [
+        .iOS(.v15)
```

Comment:
nit:

```suggestion
        .iOS(.v15),
```

as recommended earlier.

Do you understand why?

Reply:
Adding a comma allows you to explicitly indicate that this is the only element in the array and it will be easier to add new elements later.

Reply:
What does that mean "easier"?
Do you think it's hard to add a new element?

Reply:
Maybe I didn’t put it correctly, it will not be necessary to put a comma and for example, the diff will not show this line